boards/nxp/imx95_evk/doc/index.rst

```diff
@@ -233,13 +233,50 @@ To program M7, an i.MX container image ``flash.bin`` must be made, which contain
 multiple elements required, like ELE+V2X firmware, System Manager, TCM OEI, Cortex-M7
 image and so on.
 
-The steps making flash.bin and programming should refer to ``Getting Started with
+Two methods to build and program ``flash.bin``. Either can be choosed.
```

Comment:
```suggestion
Two methods to build and program ``flash.bin``. Either can be choosed.
board need set to usb serial download mode SW7 1001, and using the first serial port enumerated.
```

Reply:
I will mention the serial port information.
usb serial download mode setting had been described in method2.
For method1, doc just guides to refer to MCUX SDK doc for users who may be familar with SDK. They may be not through USB serial download.

Thanks.